keyboards/handwired/dactyl_manuform/4x6/keymaps/scheiklp/keymap.c

```diff
@@ -0,0 +1,59 @@
+/* A K.O,Y layout for the Dactyl Manuform 4x6 Keyboard */ 
+
+#include QMK_KEYBOARD_H
+#include "koy_keys_on_quertz_de_latin1.h"
+
+#define _1 0
+#define _3 1
+#define _4 2
+#define _7 3
```

Comment:
We highly recommend using an enum here, as it auto increments, and makes things easier to manage, long term. 
```suggestion
enum custom_layers {
    _1,
    _3,
    _4,
    _7,
};
```